### ISSUE TITLE
Clean up references to WP ticket numbers

### ISSUE THREAD

**Issue by nylen** [ClassicPress/ClassicPress#145]
Example: 

https://github.com/ClassicPress/ClassicPress/blob/71e29a6d71b63a9677563644d0df7dec12668790/tests/phpunit/tests/customize/nav-menus.php#L956-L959

Found via https://github.com/ClassicPress/ClassicPress/pull/144#discussion-diff-226494308L956

**Comment by senlin** [comment]
@nylen what is your suggestion to "clean up" the references? remove the `#41488` or prefix it with something like `WP Trac-`?

**Comment by Mte90** [comment]
I think that is better to leave them because they give context in case in the future we need to understand why.
Maybe is better as Pieter say with adding a prefix to differentiate them with ours

**Comment by nylen** [comment]
Yeah, I'd search for a regular expression like `\b#\d+\b` here. It won't come out perfect though, there will probably still be some other random stuff to sort through in the results.

**Comment by senlin** [comment]
Cool, if you can generate the list, then I will do the cleaning up :)

**Comment by nylen** [comment]
I find 148 items to clean up here, after removing noise like CSS colors and issues that look like they came from other projects:

```
tests/phpunit/tests/rest-api.php:530:			// Invalid dates (TODO: support parsing partial dates as ranges, see #38641)
tests/phpunit/tests/rest-api.php:562:			// Invalid dates (TODO: support parsing partial dates as ranges, see #38641)
tests/phpunit/tests/mail.php:316:		$subject = 'Testing #21659';
tests/phpunit/tests/option/siteOption.php:133:	// #15497 - ensure update_site_option will add options with false-y values
tests/phpunit/tests/option/siteOption.php:144:	// #18955 - ensure get_site_option doesn't cache the default value for non-existent options
tests/phpunit/tests/basic.php:37:			error_log( 'FIXME after PR #32 is merged' );
tests/phpunit/tests/pluggable.php:91:		// Pluggable function signatures are not tested when an external object cache is in use. #31491
tests/phpunit/tests/pluggable.php:178:		// Pluggable function signatures are not tested when an external object cache is in use. #31491
tests/phpunit/tests/media.php:329:		// #21963, there will be a guid always, so there will be a URL
tests/phpunit/tests/user/author.php:60:		// See #20285
tests/phpunit/tests/rest-api/rest-schema-setup.php:397:	 * how they were generated, see #41123.
tests/phpunit/tests/rest-api/rest-posts-controller.php:1530:			// '0000-00-00 00:00:00'.  See #38883.
tests/phpunit/tests/customize/nav-menus.php:554:		// Non-existent post types allowed as of #39610.
tests/phpunit/tests/customize/nav-menus.php:956:		// Add global namespace prefix to check #41488.
tests/phpunit/tests/customize/setting.php:376:		// Note: #29316 will allow us to have one filter for all settings of a given type, which is what we need.
tests/phpunit/tests/customize/setting.php:392:		// Note: #29316 will allow us to have one filter for all settings of a given type, which is what we need.
tests/phpunit/tests/customize/setting.php:412:		// Note: #29316 will allow us to have one filter for all settings of a given type, which is what we need.
tests/phpunit/tests/customize/setting.php:428:		// Note: #29316 will allow us to have one filter for all settings of a given type, which is what we need.
tests/phpunit/tests/customize/setting.php:447:		 * In #36952 the conditions were such that get_theme_mod() be erroneously used
tests/phpunit/tests/customize/manager.php:502:		// If there is no user, don't fetch the most recent autosave. See #42450.
tests/phpunit/tests/customize/manager.php:613:		update_option( 'posts_per_page', 1 ); // To check #39022.
tests/phpunit/tests/customize/manager.php:2528:		// See #35355.
tests/phpunit/tests/import/parser.php:193:	// tags in CDATA #11574
tests/phpunit/tests/ajax/CustomizeManager.php:527:		// Temporarily remove user to test requirement that user is logged in. See #42450.
tests/phpunit/tests/formatting/MakeClickable.php:141:	// #4570
tests/phpunit/tests/formatting/MakeClickable.php:158:	// #10990
tests/phpunit/tests/formatting/MakeClickable.php:193:	// Based on a real comments which were incorrectly linked. #11211
tests/phpunit/tests/formatting/MakeClickable.php:212:	// #14993
tests/phpunit/tests/formatting/WPTexturize.php:35:	//WP Ticket #1418
tests/phpunit/tests/formatting/WPTexturize.php:42:	//WP Ticket #3810
tests/phpunit/tests/formatting/WPTexturize.php:47:	//WP Ticket #4539
tests/phpunit/tests/formatting/WPTrimWords.php:30:	// #18726
tests/phpunit/tests/post/revisions.php:29:	 * Note: Test needs reviewing when #16215 is fixed because I'm not sure the test current tests the "correct" behavior
tests/phpunit/tests/post/revisions.php:52:		// #16215
tests/phpunit/data/export/small-export.xml:71:	<wp:category><wp:term_id>7</wp:term_id><wp:category_nicename>eternity</wp:category_nicename><wp:category_parent></wp:category_parent><wp:cat_name><![CDATA[远征手记]]></wp:cat_name><wp:category_description><![CDATA[No more duplicate categories, #13364]]></wp:category_description></wp:category>
src/wp-includes/class-wp-term-query.php:122:	 *                                                See #41796 for details.
src/wp-includes/class-wp-post-type.php:446:		// Back compat with quirky handling in version 3.0. #14122.
src/wp-includes/pluggable.php:1331:	// In php 5 parse_url may fail if the URL query part contains http://, bug #38143
src/wp-includes/script-loader.php:483:			'plural' => _n( 'There is %d error which must be fixed before you can update this file.', 'There are %d errors which must be fixed before you can update this file.', 2 ), // @todo This is lacking, as some languages have a dedicated dual form. For proper handling of plurals in JS, see #20491.
src/wp-includes/script-loader.php:589:			'plural'   => _n( 'There is %d error which must be fixed before you can save.', 'There are %d errors which must be fixed before you can save.', 2 ), // @todo This is lacking, as some languages have a dedicated dual form. For proper handling of plurals in JS, see #20491.
src/wp-includes/script-loader.php:596:			'plural'   => _n( 'Unable to save due to %s invalid setting.', 'Unable to save due to %s invalid settings.', 2 ), // @todo This is lacking, as some languages have a dedicated dual form. For proper handling of plurals in JS, see #20491.
src/wp-includes/js/admin-bar.js:33:					// so close it on a second tap and prevent propag and defaults. See #29906
src/wp-includes/js/media-views.js:7257:		 * This way, in the Media Library, it's right after the Add New button, see ticket #37188.
src/wp-includes/js/media-views.js:7271:		// For accessibility reasons, place the normal sidebar after the attachments, see ticket #36909.
src/wp-includes/js/wp-a11y.js:32:		 * See ticket #36853.
src/wp-includes/js/tinymce/plugins/wplink/plugin.js:302:		// The "de-facto standard" shortcut, see #27305
src/wp-includes/js/tinymce/plugins/wplink/plugin.js:501:						 * Looks like Safari and VoiceOver need an `aria-selected` attribute. See ticket #33301.
src/wp-includes/js/tinymce/plugins/wpeditimage/plugin.js:701:					// See #38313.
src/wp-includes/js/tinymce/skins/wordpress/wp-content.css:244:/* Fix for proprietary Mozilla display attribute, see #38757 */
src/wp-includes/js/wp-util.js:25:			 * @see trac ticket #22344.
src/wp-includes/js/customize-preview-widgets.js:50:		 * For a proposal to include attachments in the customized state, see #37887.
src/wp-includes/js/media-models.js:846:	 * See trac ticket #24753
src/wp-includes/user.php:834: * Using $strategy = 'memory' this is memory-intensive and should handle around 10^5 users, but see WP Bug #12257.
src/wp-includes/media.php:1130:	 * an incorrect image. See #35045.
src/wp-includes/media.php:1173:			// The 'src' image has to be the first in the 'srcset', because of a bug in iOS8. See #35030.
src/wp-includes/media.php:2554:			// Remove `feature` query arg and force SSL - see #40866.
src/wp-includes/media.php:2558:			// Remove all query arguments and force SSL - see #40866.
src/wp-includes/media.php:2562:			// Add loop param for mejs bug - see #40977, not needed after #39686.
src/wp-includes/media.php:3019:	// when enabled. See #29602.
src/wp-includes/media.php:3640:	// registered internally before we try to localize it. see #24724.
src/wp-includes/ms-blogs.php:785: *  - plugins. See #14941
src/wp-includes/Text/Diff.php:50:        // WP #7391
src/wp-includes/Text/Diff/Renderer/inline.php:16:// WP #7391
src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php:1419:			// #38883).  In this case, shim the value based on the `post_date`
src/wp-includes/customize/class-wp-customize-new-menu-control.php:8: * @deprecated WP-4.9.0 This file is no longer used as of the menu creation UX introduced in #40104.
src/wp-includes/customize/class-wp-customize-new-menu-control.php:15: * @deprecated WP-4.9.0 This class is no longer used as of the menu creation UX introduced in #40104.
src/wp-includes/customize/class-wp-customize-new-menu-control.php:39:		_deprecated_file( basename( __FILE__ ), 'WP-4.9.0' ); // @todo Move this outside of class in 5.0, and remove its require_once() from class-wp-customize-control.php. See #42364.
src/wp-includes/customize/class-wp-customize-new-menu-section.php:8: * @deprecated WP-4.9.0 This file is no longer used as of the menu creation UX introduced in #40104.
src/wp-includes/customize/class-wp-customize-new-menu-section.php:15: * @deprecated WP-4.9.0 This class is no longer used as of the menu creation UX introduced in #40104.
src/wp-includes/customize/class-wp-customize-new-menu-section.php:41:		_deprecated_file( basename( __FILE__ ), 'WP-4.9.0' ); // @todo Move this outside of class in 5.0, and remove its require_once() from class-wp-customize-section.php. See #42364.
src/wp-includes/css/admin-bar.css:612:/* The admin bar search field needs to reset many styles that might be inherited from the active Theme CSS. See ticket #40313. */
src/wp-includes/css/editor.css:783:/* Don't show the tooltip. Used in Chrome RTL, see #42018 */
src/wp-includes/theme.php:1800:			// Trigger creation of a revision. This should be removed once #30854 is resolved.
src/wp-includes/theme.php:3069: * editing flow in the Customizer. See #39752.
src/wp-includes/formatting.php:342:						// This is most likely to be problematic in the context of bug #18549.
src/wp-includes/formatting.php:4938: * This is designed to replace the PCRE \s sequence.  In ticket #22692, that
src/wp-includes/class-wp-http-ixr-client.php:66:		// Merge Custom headers ala #8145
src/wp-includes/class-wp-customize-widgets.php:709:		 * Once JS i18n is implemented (in #20491), this can be removed.
src/wp-includes/IXR/class-IXR-client.php:71:        // Merged from WP #8145 - allow custom headers
src/wp-includes/IXR/class-IXR-client.php:117:            	// merged from WP #12559 - remove trim
src/wp-includes/IXR/class-IXR-server.php:43:                    status_header( 405 ); // WP #20986
src/wp-includes/IXR/class-IXR-server.php:46:                header('Content-Type: text/plain'); // merged from WP #9093
src/wp-includes/class-wp-customize-manager.php:320:		require_once( ABSPATH . WPINC . '/customize/class-wp-customize-new-menu-control.php' ); // @todo Remove in 5.0. See #42364.
src/wp-includes/class-wp-customize-manager.php:328:		require_once( ABSPATH . WPINC . '/customize/class-wp-customize-new-menu-section.php' ); // @todo Remove in 5.0. See #42364.
src/wp-includes/class-wp-customize-manager.php:1191:			 * to get re-used. See #40146.
src/wp-includes/class-wp-customize-manager.php:2824:		$json_options |= JSON_PRETTY_PRINT; // Also introduced in PHP 5.4, but WP defines constant for back compat. See WP Trac #30139.
src/wp-includes/class-wp-customize-manager.php:3054:	 * This should be able to be removed once #40922 is addressed in core.
src/wp-includes/class-wp-customize-manager.php:4714:		// Temporarily disable installation in Customizer. See #42184.
src/wp-includes/IXR/class-IXR-message.php:53:        // merged from WP #10698 - this method avoids the RAM usage of preg_replace on very large messages
src/wp-includes/template-loader.php:18: * content loads for 'HEAD' requests. See #14348.
src/wp-includes/class-wp-image-editor-gd.php:45:		// On some setups GD library does not provide imagerotate() - Ticket #11536
src/wp-includes/load.php:694:	// Use parens for clone to accommodate PHP 4. See #17880
src/wp-includes/class-json.php:279:        // see bug #16908 - regarding numeric locale printing
src/wp-includes/class-wp-http-curl.php:139:		 * a bug #17490 with redirected POST requests, so handle redirections outside Curl.
src/wp-includes/rewrite.php:417:	// Bug found in #11694 - 'page' was returning '/4'
src/wp-includes/class-wp-xmlrpc-server.php:5215:					// This method used to omit the trailing new line. #23219
src/wp-includes/widgets/class-wp-widget-custom-html.php:219:				'plural' => _n( 'There is %d error which must be fixed before you can save.', 'There are %d errors which must be fixed before you can save.', 2 ), // @todo This is lacking, as some languages have a dedicated dual form. For proper handling of plurals in JS, see #20491.
src/wp-includes/widgets/class-wp-widget-recent-comments.php:52:		if ( ! current_theme_supports( 'widgets' ) // Temp hack #14876
src/wp-admin/user-edit.php:574:		<input class="hidden" value=" " /><!-- #24364 workaround -->
src/wp-admin/js/tags-suggest.js:170:			// Looks like Safari and VoiceOver need an `aria-selected` attribute. See ticket #33301.
src/wp-admin/js/plugin-install.js:153:	 * with other handlers, see Trac ticket #43082.
src/wp-admin/js/common.js:507:	 * that are (incorrectly) using it. Do not use. Use `.inline` instead. See #34570.
src/wp-admin/js/common.js:543:	// This event needs to be delegated. Ticket #37973.
src/wp-admin/js/common.js:574:	// This event needs to be delegated. Ticket #37973.
src/wp-admin/js/common.js:682:			// Reset paging var for new filters/searches but not for bulk actions. See #17685.
src/wp-admin/js/customize-nav-menus.js:3114:		new_menu: api.Menus.NewMenuControl, // @todo Remove in 5.0. See #42364.
src/wp-admin/js/customize-controls.js:2565:			// Temporary special function since supplying SFTP credentials does not work yet. See #42184.
src/wp-admin/js/customize-controls.js:3076:			// Temporary since supplying SFTP credentials does not work yet. See #42184
src/wp-admin/js/customize-controls.js:3199:			// Temporary since supplying SFTP credentials does not work yet. See #42184.
src/wp-admin/js/customize-controls.js:3413:			// Temporary since supplying SFTP credentials does not work yet. See #42184.
src/wp-admin/js/customize-controls.js:3538:			control.selector = '#customize-control-' + id.replace( /\]/g, '' ).replace( /\[/g, '-' ); // Deprecated, likely dead code from time before #28709.
src/wp-admin/js/customize-controls.js:3542:				control.container = $( control.selector ); // Likely dead, per above. See #28709.
src/wp-admin/js/customize-controls.js:5133:			// Temporary special function since supplying SFTP credentials does not work yet. See #42184.
src/wp-admin/js/customize-controls.js:5171:					// Temporary special function since supplying SFTP credentials does not work yet. See #42184.
src/wp-admin/js/customize-controls.js:6323:			previewFrame.iframe.attr( 'onmousewheel', '' ); // Workaround for Safari bug. See WP Trac #38149.
src/wp-admin/js/postbox.js:282:			// Return on the nav-menus.php screen, see #35112.
src/wp-admin/js/widgets.js:162:					 * it's important for screen readers, see ticket #31476.
src/wp-admin/js/widgets.js:171:					 * it's important for screen readers, see ticket #31476.
src/wp-admin/js/edit-comments.js:292:				e.stopPropagation(); // ticket #35904
src/wp-admin/js/edit-comments.js:321:			undoing = false; // ticket #35904
src/wp-admin/js/nav-menu.js:845:			 * in a single input to avoid PHP `max_input_vars` limitations. See #14134.
src/wp-admin/js/nav-menu.js:881:			 * in legacy browsers. See changeset 34078, see also ticket #26600#comment:59
src/wp-admin/js/nav-menu.js:1108:			 * links thus excluding the current page `<span>`. See ticket #35577.
src/wp-admin/js/image-edit.js:57:		 * `el` can be a single form element or a fieldset. Before #28864, the disabled state on
src/wp-admin/js/widgets/media-gallery-widget.js:111:			 * For a proposal to include attachments in the customized state, see #37887.
src/wp-admin/js/widgets/media-widgets.js:118:				// This should be eliminated once #40450 lands of when this is merged into core.
src/wp-admin/nav-menus.php:54: * into `$_POST` to avoid PHP `max_input_vars` limitations. See #14134.
src/wp-admin/css/color-picker.css:52:	/* See Trac ticket #39662 */
src/wp-admin/css/list-tables.css:861:/* ticket #34150 */
src/wp-admin/css/customize-nav-menus.css:308:/* rework the arrow indicator implementation for NVDA bug same as #32715 */
src/wp-admin/css/customize-nav-menus.css:515:	/* Reset the value inherited from the base .accordion-section-title style. Ticket #37589. */
src/wp-admin/css/customize-nav-menus.css:527:/* rework the arrow indicator implementation for NVDA bug see #32715 */
src/wp-admin/css/customize-widgets.css:435:	/* Compact widget-tops on smaller laptops, but not tablets. See ticket #27112#comment:4 */
src/wp-admin/css/common.css:2846:/* For non-js plugin installation screen ticket #36430. */
src/wp-admin/css/customize-controls.css:2858:	   bring it back see ticket #35220 */
src/wp-admin/customize.php:52:		 * By opening an Ajax request, this is avoided and the changeset is published. See #39221.
src/wp-admin/user-new.php:446:			<input class="hidden" value=" " /><!-- #24364 workaround -->
src/wp-admin/includes/nav-menu.php:1098: * it into `$_POST` to avoid PHP `max_input_vars` limitations. See #14134.
src/wp-admin/includes/class-wp-community-events.php:442:	 * @deprecated WP-4.9.0 Use a plugin instead. See #41217 for an example.
src/wp-admin/includes/media.php:1889:// when enabled. See #29602.
src/wp-admin/includes/upgrade.php:1768:	// Remove plugins with callback as an array object/method as the uninstall hook, see #13786.
src/wp-admin/includes/image-edit.php:187:		// On some setups GD library does not provide imagerotate() - Ticket #11536
src/wp-admin/includes/deprecated.php:1473: * when the user navigates to it with the browser's Back button. See #28037
src/wp-admin/includes/post.php:1244:	// Note: if empty name is supplied -- use the title instead, see #6072
src/wp-admin/includes/update-core.php:601:// Following files added back in 4.5 see #36083
src/wp-admin/includes/update-core.php:616:// Added back in 4.9 [41328], see #41755
src/wp-admin/includes/update-core.php:708:'wp-admin/includes/class-wp-automatic-upgrader.php', // Wrong file name, see #37628.
src/wp-admin/includes/ms-deprecated.php:20:	// Deprecated. See #11763.
```

**Comment by senlin** [comment]
Great thanks for the list I will work on it later today!

**Comment by nylen** [comment]
@senlin all of the recent PRs are merged, so I think we are done with this, correct?

**Comment by senlin** [comment]
@nylen Yes Sir, closing this issue.
